_dashboards/management/data-sources.md

```diff
@@ -19,61 +20,33 @@ Once you have installed OpenSearch and OpenSearch Dashboards, you can use Dashbo
-
-    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/delete-data-source.png" alt="Deleting a data source UI" width="700"/>
-
-### Modify a data source connection
-
-To make changes to a data source connection, select a connection in the list on the **Data sources** main page. The **Connection Details** window opens.
-
-To make changes to **Connection Details**, edit one or both of the **Title** and **Description** fields and select **Save changes** in the lower-right corner of the screen. You can also cancel changes here. To change the **Authentication Method**, choose a different authentication method, enter your credentials (if applicable), and then select **Save changes** in the lower-right corner of the screen. The changes are saved.
-
-When **Username & Password** is the selected authentication method, you can update the password by choosing **Update stored password** next to the **Password** field. In the pop-up window, enter a new password in the first field and then enter it again in the second field to confirm. Select **Update stored password** in the pop-up window. The new password is saved. Select **Test connection** to confirm that the connection is valid.
+## Permissions
 
-When **AWS SigV4** is the selected authentication method, you can update the credentials by selecting **Update stored AWS credential**. In the pop-up window, enter a new access key in the first field and a new secret key in the second field. Select **Update stored AWS credential** in the pop-up window. The new credentials are saved. Select **Test connection** in the upper-right corner of the screen to confirm that the connection is valid.
+To work with data sources in OpenSearch Dashboards, make sure that the user has been assigned the correct cluster-level [data source permission]({{site.url}}{{site.baseurl}}/security/access-control/permissions#data-source-permissions).
 
-To delete the data source connection, select the delete icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/dashboards/trash-can-icon.png" class="inline-icon" alt="delete icon"/>{:/}).
+## Types of data streams
 
-## Create an index pattern
+There are two ways to configure data sources
 
-Once you've created a data source connection, you can create an index pattern for the data source. An _index pattern_ is a template that OpenSearch uses to create indexes for data from the data source. See [Index patterns]({{site.url}}{{site.baseurl}}/dashboards/management/index-patterns/) for more information and a tutorial. 
+1. **Management** -> **Dashboards Management** -> **Data sources**
+    
+    Can be used for OpenSearch Data Stream Connections, further details are available at [Configuring and using multiple data sources]({{site.url}}{{site.baseurl}}/dashboards/management/multi-data-sources/)
+2. **Management** -> **Data sources** 
```

Comment:
I don't know if this is accurate anymore, since I think these two pages merged?

Reply:
we have merged the experience in 2.16. So this is correct.

Reply:
Sorry for the confusion, @AntonEliatra! They DID make changes in 2.16 to the UI. Since the release is today (8/7), you should be able to see the updates tomorrow in the playground for testing purposes. In addition, @RyanL1997 is going to capture a video of the new flow and add it to a doc issue. 

We'll have to update the rest of documentation for the new changes post-2.16.

Reply:
@RyanL1997 I don't see the option Management > Data sources in my local environment running 2.16. I only have the option to go through Dashboards Management. Please let me know if Management > Data sources is available in 2.16.